.github/workflows/publish-prod-images.yml

```diff
@@ -72,10 +72,10 @@ jobs:
           uses: docker/build-push-action@v3
           continue-on-error: true
           with:
-              context: .
-              file: tools/sigclient/Dockerfile
+              context: tools/sigclient
+              file: Dockerfile
               push: true
-              tags: sigclient/sigclient:latest
+              tags: siglens/sigclient:latest
```

Comment:
I'm not sure we want this change. @nkunal what do you think?

Reply:
yeah actually the current one fails at build and push, cause it did not find the username `sigclient`. But if we want to keep `sigclient` separate from SigLens. Then probably we should create a new username.